app/src/include/firebase/app.h

```diff
@@ -701,6 +701,23 @@ class App {
   static void SetDefaultConfigPath(const char* path);
 #endif  // INTERNAL_EXPERIMENTAL
 
+#ifdef INTERNAL_EXPERIMENTAL
+  // These methods are only visible to SWIG and internal users of firebase::App.
+
+  // On Desktop, log a Heartbeat
```

Comment:
uber nit: use three slashes. And those in line 710 and 715

Reply:
Done. Also, to see if I am understanding correctly, does triple slash indicate docstring comment whereas double slash is a comment on implementation details?